### PR TITLE
Add Tutorial: on how to configure k8s authentication with load balancer

### PR DESCRIPTION
**Pull request by bka**
I have read and understood the Contributor's Certificate of Origin available at the end of 
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Bernhard <bl@wamoco.de>

### CODE REVIEW

File: tutorials/howto-k8s-authentication-with-load-balancer/01.en.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+author_img: "https://avatars3.githubusercontent.com/u/584644"
+author_description: "Magento and Web Developer Fullstack, Co-Founder Wamoco"
+language: "en"
+available_languages: ["en", "de"]
+header_img: "header-4"
+---
+
+## Introduction
+
+We installed a Kubernetes Cluster on [Hetzner Cloud](https://www.hetzner.com/cloud) and used its [Cloud Load Balancer](https://www.hetzner.com/de/cloud/load-balancer) to serve traffic. After some experimentation, we wanted to protect our deployed services with some authentication. Adding [basic auth](https://en.wikipedia.org/wiki/Basic_access_authentication) was no problem. But as soon as we tried to bypass some source adresses, it wasn´t working.
```
Comment (rook59) [inline tutorials/howto-k8s-authentication-with-load-balancer/01.en.md:20]: > https://www.hetzner.com/de/cloud/load-balancer

German link in english tutorial

File: tutorials/howto-k8s-authentication-with-load-balancer/01.en.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    data:
+      use-forwarded-headers: "true"
+      compute-full-forwarded-for: "true"
+      use-proxy-protocol: "true"
+
+### Step 1.1 - Create and connect the Load Balancer
+
+Create a Load Balancer in your hetzner cloud console. It is important to select the correct internal network. We did not need to define any services at this step, this was done automatically after deploying the ingress service.
+
+![PHP Info Page](images/loadbalancer-creation.png)
```
Comment (rook59) [inline tutorials/howto-k8s-authentication-with-load-balancer/01.en.md:64]: PHP Info Page? :D Probably copy&paste mistake

File: tutorials/howto-k8s-authentication-with-load-balancer/01.de.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
+---
+SPDX-License-Identifier: MIT
+path: "/tutorials/howto-k8s-authentication-with-load-balancer"
```
Comment (rook59) [inline tutorials/howto-k8s-authentication-with-load-balancer/01.de.md:3]:
```suggestion
path: "/tutorials/howto-k8s-authentication-with-load-balancer/de"
```

**Review by rook59 — COMMENTED** [review]
Thank you for your submission. Please take a look at my notes.

**Comment by rook59** [comment]
@bka can you fix those two small issues? afterwards I guess the tutorial is ready

File: tutorials/howto-k8s-authentication-with-load-balancer/01.en.md (hunk 2 above)
Comment (bka) [reply tutorials/howto-k8s-authentication-with-load-balancer/01.en.md]: jep

**Comment by bka** [comment]
@rook59 thanks for review, and sorry for the delay, addressed all your comments

**Review by rook59 — APPROVED** [review]
LGTM

**Pull request merged**